onnx/test/reference_evaluator_loop_test.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) ONNX Project Contributors
+
+# SPDX-License-Identifier: Apache-2.0
+# type: ignore
+import unittest
+
+import numpy as np
+
+from onnx import TensorProto
+from onnx.helper import (
+    make_graph,
+    make_model,
+    make_node,
+    make_opsetid,
+    make_tensor_value_info,
+)
+from onnx.numpy_helper import from_array
+from onnx.reference import ReferenceEvaluator
+
```

Comment:
Import modules only

Reply:
Not all files are following this rule yet. I did not want to add changes not related to the issue the PR fixes.

Reply:
This file is all new code so might be good to start it following the convention?

Reply:
True... My mistake. I split it from an existing file.